metricbeat/module/ceph/_meta/Dockerfile

```diff
@@ -11,3 +8,8 @@ HEALTHCHECK --interval=1s --retries=300 \
         | jq .output.health.health_services[0].mons[0] \
         | grep health
 EXPOSE 5000
+
+ENV NETWORK_AUTO_DETECT 4
```

Comment:
I couldn't make it work with static config as we had before, but it seems to work with this.